main.py

```diff
@@ -167,6 +167,12 @@ def campaign_group_page(campaign_group=None, pwg=None):
     conn = sqlite3.connect('data.db')
     cursor = conn.cursor()
     sql_args = [campaign_group]
+    counts = cursor.execute("SELECT COUNT(*) FROM samples WHERE campaign_group = ?", sql_args)
+    counts = counts.fetchall()
+    counts = int(counts[0][0])
+    page = int(request.args.get('page', 1))
+    page_size = 10
+    num_pages = counts/page_size + 1
```

Comment:
This fails if `counts` divide by `page_size`, e.g.
```
page_size = 10
counts = 20
num_pages = 20 / 10 + 1 = 3
```
But there should be only 2 pages.
```
from math import ceil
num_pages = ceil(float(counts) / page_size)
```
`ceil` is a "ceiling" function that rounds up. If we round up 20/10, we still get 2.